code/arith.py

```diff
@@ -9,7 +9,7 @@ def main():
     try:
         operand1, operand2 = float(sys.argv[2]), float(sys.argv[3])
     except ValueError:
-        print 'cannot convert input to a number: bailing out'
+        print ('cannot convert input to a number: bailing out')
```

Comment:
May want to remove the space between `print` and `(` to comply with PEP8.

Reply:
I did this as part of the merge (including fixing other PEP8 issues).